package.nls.json

```diff
@@ -59,6 +59,7 @@
   "configuration.SAS.results.html.style.(server default)": "Default to the style configured on the SAS server.",
   "configuration.SAS.results.sideBySide": "Display results to the side of the code",
   "configuration.SAS.results.singlePanel": "Reuse single panel to display results",
+  "configuration.SAS.log.parseOutputLog.enabled": "Enable/disable parse output log from running sas file add show problems",
```

Comment:
nit: the description should be from end user's perspective. "Parse" is an implentation perspective. Perhaps say something like
"Show problems from SAS log"